cosmos.gemspec

```diff
@@ -73,12 +73,10 @@ spec = Gem::Specification.new do |s|
       files
     else [] end
 
-  s.has_rdoc = true
-
-  s.required_ruby_version = '~> 2.2'
+  s.required_ruby_version = '~> 2.3'
```

Comment:
I think it is fine to stop testing 2.2 just for performance of the tests, but is there actually a reason?   If not we should probably keep support at the .gemspec level until someone tells us that it doesn't work.  thoughts?

Reply:
Do other dependency changes below justify removing 2.2?  Also are the changes required?

Reply:
Bundler 2.x requires Ruby >= 2.3.0 so I can revert this because COSMOS doesn't require 2.3 or later but it'll be more difficult to install on Ruby 2.2 because you'll have to install an old version of Bundler.

Reply:
It looks like the latest versions of nokogiri and reek are now requiring 2.3. Nokogiri is especially annoying because bundler wants to get the latest. I think it's time to say good bye to Ruby 2.2. All support ended March 31, 2018 and no bugs or security fixes are being back-ported.